build.sbt

```diff
@@ -20,11 +20,15 @@ scalacOptions := Vector(
   "-Dscala.usejavacp=true"
 )
 
+// unofficial signal-service-java repo
+resolvers += Resolver.url("custom signal-service-java repo", url("https://raw.github.com/AsamK/maven/master/releases/"))
```

Comment:
no need for that, the jar is in [maven central](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.github.turasa%22)